### DISCUSSION TITLE
Using led PSU to feed rpi.

### DISCUSSION THREAD

**Discussion by Pausejo** [onlaj/Piano-LED-Visualizer#299] [Q&A]
Hi there, I recently build the piano led visualizer and I realised that there are lots of wires going in and out the rpi. Would it be possible to avoid the micro usb power input and supply directly through the 5v 8 amp PSU?

As long as I've searched, the rpi will only take the current that it needs.

**Comment by jwessel** [comment]
https://raspberrypi.stackexchange.com/questions/61302/does-raspberry-pi-zero-have-a-polyfuse   

The RPI Zero has no fuse circuitry.  The 5 volt and ground connections are wired straight through from the USB connector to the 40 pin connector.   This means you hook it up however you see fit.  Just don't mix up the connections. :-)
